music21/stream/__init__.py

```diff
@@ -8552,6 +8528,10 @@ def bestMatch(target, divisors):
                             and signedError != 0):
                         e.editorial.quarterLengthQuantizationError = signedError
 
+            # end for e in ._elements
+            # ran coreSetElementOffset
+            useStream.coreElementsChanged()
```

Comment:
thanks!  can include `updateIsFlat=False` (but not isSorted, since the quantization might move elements.